sdk/core/azure-core-http-okhttp/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Updated OkHttpClientBuilder to disallow OkHttpClient to follow redirects by default. This gives more control to the pipeline to add redirect policies as needed.
```

Comment:
Please add explicit pointers how users would get redirect back, i.e. pointer to redirect policy and option to provide own okhttpclient to the builder.